editors/vscode/src/extension.ts

```diff
@@ -202,6 +202,9 @@ function startClient(context: vscode.ExtensionContext) {
 }
 
 export function activate(context: vscode.ExtensionContext) {
+    if (!!process.env.CODESPACES) {
```

Comment:
```suggestion
    if (process.env.hasOwnProperty("CODESPACES")) {
```

I find that more readable than the double negation :)

There's also `"CODESPACES" in process.env` and a few others.